mscore/editstyle.ui

```diff
@@ -9354,7 +9391,7 @@
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="extensionAdjust">
                 <property name="minimum">
-                 <double>-99.99</double>
+                 <double>-99.989999999999995</double>
```

Comment:
why is it suddenly needed?

Reply:
It's automatically saved that way in newer versions of QtCreator. It probably is not needed, but clearly does not harm.

Reply:
I can manually remove this to simplify the diff.